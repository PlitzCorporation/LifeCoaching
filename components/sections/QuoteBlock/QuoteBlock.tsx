import { WhiteThickLine } from '@/components/ui/decorations';
import React from 'react';

const QuoteBlock = () => {
	return (
		<div className="bg-gradient-to-b from-plitz-accent to-plitz-secondary py-20 lg:py-40">
			<div className=" max-w-screen-xl plitz-container drop-shadow-plitz-darker text-white text-balance">
				<div className="w-max mx-auto mb-8">
					<WhiteThickLine />
				</div>
				<p className="text-center text-2xl lg:text-4xl font-plitz-font-main font-bold">
					My mission is to help uncover the core, the thing driving behavior and
					choices that have been imprinted in your life. What is the unmet
					emotional need that keeps you in a scramble? If you don’t understand
					or not aware, you won’t be empowered to flourish.
				</p>
			</div>
		</div>
	);
};

export default QuoteBlock;
