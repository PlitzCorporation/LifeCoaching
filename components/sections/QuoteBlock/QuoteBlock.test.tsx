import { render } from '@testing-library/react';
import QuoteBlock from './QuoteBlock';

describe('QuoteBlock', () => {
	it('should render correctly', () => {
		const { container } = render(<QuoteBlock />);
		expect(container.firstChild).toMatchSnapshot();
	});
});
