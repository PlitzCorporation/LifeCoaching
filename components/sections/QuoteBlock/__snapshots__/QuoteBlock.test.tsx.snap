// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteBlock should render correctly 1`] = `
<div
  class="bg-gradient-to-b from-plitz-accent to-plitz-secondary py-20 lg:py-40"
>
  <div
    class=" max-w-screen-xl plitz-container drop-shadow-plitz-darker text-white text-balance"
  >
    <div
      class="w-max mx-auto mb-8"
    >
      <div
        class="block w-32 h-1 bg-gradient-to-r from-white/5 to-white/100"
      />
    </div>
    <p
      class="text-center text-2xl lg:text-4xl font-plitz-font-main font-bold"
    >
      My mission is to help uncover the core, the thing driving behavior and choices that have been imprinted in your life. What is the unmet emotional need that keeps you in a scramble? If you don’t understand or not aware, you won’t be empowered to flourish.
    </p>
  </div>
</div>
`;
